### PR TITLE
Catalog view updates

### PR DESCRIPTION
**Pull request by wendellpiez**
This includes several improvements in XSLT for HTML and FO-based PDF production, making them more robust, more featureful and more general.

Includes scripts and a `pom.xml` file for invoking the production pipeline in Maven.

When these are tested and documented this should be ready.

### CODE REVIEW

**Review by aj-stein-nist — APPROVED** [review]
Minor non-blocking feedback, but nice work. Thanks.

File: publish/nist-emulation/oscal_sp800-53-emulator_fo.xsl [hunk 1]
```diff
@@ -67,7 +78,7 @@
           <rdf:RDF xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#">
             <rdf:Description rdf:about="" xmlns:dc="http://purl.org/dc/elements/1.1/">
               <dc:title>
-                <xsl:value-of select="/html/head/title"/>
+                <xsl:call-template name="doctitle-string"/>
               </dc:title>
               <!--<dc:creator> XOXOXOXOX </dc:creator>
               <dc:description>  XOXOXOXOX </dc:description>-->
```
[the remark this answers is not in the document]
Comment (wendellpiez) [reply publish/nist-emulation/oscal_sp800-53-emulator_fo.xsl:84]: Actually this is probably the one comment I would leave in ... I can add explanation why.

[the remark this answers is not in the document]
Comment (wendellpiez) [reply publish/nist-emulation/oscal_sp800-53-emulator_fo.xsl]: See https://github.com/usnistgov/oscal-xslt/pull/4/commits/53d7aa0b4917571f2608a1e419b1dfb9515179e9 for some fix up.

**Comment by aj-stein-nist** [comment]
@Arminta-Jenkins-NIST and @nikitawootten-nist, I would like if you can review this before the end of Sprint 67. Please pull down the code and [the README documentation here](https://github.com/usnistgov/oscal-xslt/blob/c6e6e0265a33a2698dbef8cc0c3a466d188f27ff/publish/readme.md). Thanks!

**Comment by aj-stein-nist** [comment]
Not completely related so I put it in the OSCAL DKB wiki as an entry, but I added information about how to set up Oxygen transformations in the style I would if you will not exclusively rely on the Maven shell script.

https://github.com/usnistgov/OSCAL/wiki/Developer-Knowledge-Base/5810ad9a2d965fc526995d53ccb91ac129958b69

**Review by nikitawootten-nist — CHANGES_REQUESTED** [review]
B: Please mark the scripts in `publish/` as executable by running:

```bash
chmod +x publish/*.sh
```

**Comment by nikitawootten-nist** [comment]
Hi Wendell, I'm also having trouble running the PDF generation script:

```bash
 ➜ ./mvn-make-catalog-pdf.sh basic-catalog.xml result.pdf
[INFO] Scanning for projects...
[INFO]
[INFO] -------< gov.nist.secauto.oscal.tools.core:oscal-xml-formatting >-------
[INFO] Building oscal-xml-formatting 1.0.0
[INFO]   from pom.xml
[INFO] --------------------------------[ pom ]---------------------------------
[INFO]
[INFO] --- exec:3.1.0:java (default-cli) @ oscal-xml-formatting ---
Apr 24, 2023 4:02:09 PM org.apache.fop.apps.FOUserAgent processEvent
INFO: Rendered page #1.
Apr 24, 2023 4:02:09 PM org.apache.fop.apps.FOUserAgent processEvent
WARNING: The contents of fo:region-after on page i exceed the available area in the block-progression direction by 4800 millipoints. (See position 1:4091)
Apr 24, 2023 4:02:09 PM org.apache.fop.apps.FOUserAgent processEvent
WARNING: The following feature isn't implemented by Apache FOP, yet: table-layout="auto" (on fo:table) (See position 1:8458)
Error on line 1 column 10474 of basic-catalog.xml:
  SXCH0003  org.apache.fop.fo.ValidationException: "fo:page-sequence" is not a valid child
  of "fo:block"! (See position 1:10474). Caused by org.apache.fop.fo.ValidationException:
  file:/Users/naw2/Documents/repos/oscal/oscal-xslt/publish/basic-catalog.xml:1:10474:
  "fo:page-sequence" is not a valid child of "fo:block"! (See position 1:10474)
[com.xmlcalabash.drivers.Main.main()] ERROR com.xmlcalabash.util.DefaultXProcMessageListener - render-oscal-catalog.xpl:52:67:Failed to process FO document with FOP
[com.xmlcalabash.drivers.Main.main()] ERROR com.xmlcalabash.util.DefaultXProcMessageListener -     cause: file:/Users/naw2/Documents/repos/oscal/oscal-xslt/publish/basic-catalog.xml:1:10474:err:SXCH0003:org.apache.fop.fo.ValidationException: "fo:page-sequence" is not a valid child of "fo:block"! (See position 1:10474)
[com.xmlcalabash.drivers.Main.main()] ERROR com.xmlcalabash.drivers.Main - Failed to process FO document with FOP
[com.xmlcalabash.drivers.Main.main()] ERROR com.xmlcalabash.drivers.Main - Underlying exception: org.apache.fop.fo.ValidationException: "fo:page-sequence" is not a valid child of "fo:block"! (See position 1:10474)
```

**Comment by wendellpiez** [comment]
@nikitawootten-nist @Arminta-Jenkins-NIST feel free to try again thanks!

(The `basic-catalog.xml` isn't the prettiest thing ever but at least it should run now.)

**Comment by nikitawootten-nist** [comment]
The new update does fix the issue triggered by generating a PDF on a basic catalog!

**Review by nikitawootten-nist — APPROVED** [review]

**Comment by nikitawootten-nist** [comment]
@wendellpiez is this ready to merge now?

**Pull request merged**